package.json

```diff
@@ -26,12 +26,13 @@
   },
   "homepage": "https://github.com/pawelgalazka/runjs#readme",
   "dependencies": {
-    "babel": "5.8.23",
     "chalk": "1.1.0",
     "chokidar": "1.2.0",
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
+    "pkg-config": "^1.1.0"
   },
   "devDependencies": {
+    "babel": "5.8.23",
```

Comment:
They'd have to install it. Look at it the other way: what if a user already has babel installed? With babel as a direct dependency that could only use babel 5 out of the box. What if instead they really just wanted to use coffeescript or typescript? It would be a bit wasteful to install babel then. 

This change really gives the user the ability to use whichever babel version they want without configuring it. They just need to install their babel version as a dependency/devDependency of their current project.